Mogakco/Sources/Data/DataSources/Remote/ChatDataSource.swift

```diff
@@ -9,38 +9,86 @@
 import RxSwift
 import Firebase
 
-struct ChatDataSource: ChatDataSourceProtocol {
+final class ChatDataSource: ChatDataSourceProtocol {
+    
+    var listener: ListenerRegistration?
+    var page: DocumentSnapshot!
     
     enum Collection {
         static let ChatRoom = Firestore.firestore().collection("chatroom")
```

Comment:
```suggestion
        static let chatRoom = Firestore.firestore().collection("chatroom")
```
코드 컨벤션상 소문으로 시작하면 좋을 것 같아요!